src/lightning/app/core/queues.py

```diff
@@ -291,6 +291,7 @@ def get(self, timeout: Optional[int] = None):
         if timeout is None:
             # this means it's blocking in redis
             timeout = 0
+        # ToDo this is strange logic
```

Comment:
It's not as strange as it looks like. The comment above 'this means it's blocking in redis' could maybe be a bit more descriptive, but it means we need to translate the edge cases for redis. In our api, timout=None means no timeout and the queue is blocking. In redis however, timeout=0 means blocking.

Reply:
```suggestion
```